PaymentValidator.js

```diff
@@ -27,28 +27,28 @@ class PaymentValidator {
   }
 
   validate(payment) {
-    const errors = [];
+    const errors = {};

```

Comment:
nah, 
Validator isn't responsible for the whole response object 
only validation errors.
so this thing should be kept as an array of errors.